config/cesm/config_grids.xml

```diff
@@ -110,6 +110,12 @@
       <grid name="rof">null</grid>
     </model_grid>
 
+    <model_grid alias="1x1_NEON" compset="DATM.+CLM|DATM.+SLND">
```

Comment:
It appears that 1x1_NEON is doing the same thing as CLM_USRDAT_NAME, I'm not sure we need an additional name here that's specific for NEON.

Reply:
I'd also suggest when using 1x1_ to use 1x1pt_ as used in other places since the default units for grids are in degrees, and the "pt" says that in this case it's "points". So 1x1pt_ is a single point regional grid, while something like 5x5pt_ is a grid that's has 5 points in latitude and also 5 in longitude.

Reply:
If you look at the line below you will see the reason for this name, it supports all of the NEON tower sites:
<file grid="atm|lnd">$DIN_LOC_ROOT/share/domains/single_point/datmdata_NEON/$NEONSITE/domain.lnd.fv\
0.9x1.25_gx1v7_$NEONSITE.nc</file>

How about we change the file name to:  domain.lnd.1x1pt_$NEONSITE_20210210.nc

Reply:
Yes, since these files shouldn't change (except maybe adding new sites?) I think that solution makes perfect sense.

Reply:
The one other question I had about this though is I though @mvertens was doing something where single point sites didn't require domain files anymore for NUOPC? I'd have to look at it again to see how that worked though...

Reply:
OK, I looked at that again, and what she was doing was to eliminate the need for both mesh files AND domain files. But, it's still configured to need domain files.